model/Makefile

```diff
@@ -12,4 +12,9 @@ docs/pyrenew_demo.md: docs/pyrenew_demo.qmd
 docs/getting-started.md: docs/getting-started.qmd
 	quarto render docs/getting-started.qmd
 
-.PHONY: install test docs
+clean:
+	rm -rf docs/*_files/
+	rm -f docs/getting-started.ipynb
+	rm -f docs/pyrenew_demo.ipynb
```

Comment:
The thing is, when using jupyter as quarto engine, it generates the notebooks, which are only deleted once the document is successfully created. Otherwise, if it fails during compilation, the notebook is kept in there. The problem is that it is sort of used as cache, which is something we don't want it if failed! That said, I'll see if quarto has further options for the jupyter engine to avoid that behavior.

Reply:
Ah I see. That makes sense.

Reply:
Just checked, there aren't many options to play with in [quarto+jupyter](https://quarto.org/docs/computations/execution-options.html#jupyter-options). So I'll mark this as resolved.